icq-relayer/pkg/runner/run.go

```diff
@@ -719,9 +719,9 @@ func prepareMessages(msgSlice []Message, logger log.Logger) []sdk.Msg {
 	}
 
 	for _, entry := range msgSlice {
-		//fmt.Println("prepareMessages", idx)
+		// fmt.Println("prepareMessages", idx)
 		if len(list) > TxMsgs {
-			//fmt.Println("transaction full; requeueing")
+			// fmt.Println("transaction full; requeueing")
 			go func(entry Message) { time.Sleep(time.Second * 2); sendQueue <- entry }(entry) // client update not ready; requeue.
 			continue
 		}
```

Comment:
should we move commented  print statements to some logs for debugging if needed?

Reply:
Yes, I think debug logs are a better idea. But I feel this should be done as a separate issue, else this PR is going to get noisy (-er).